pages/dkp/konvoy/2.2/intro-dkp/index.md

```diff
@@ -9,7 +9,7 @@ enterprise: false
 ---
 As the leading independent platform for Kubernetes in production, the D2iQ Kubernetes Platform (DKP) provides a holistic approach and a complete set of enterprise-grade technologies, services, training, and support to build and run applications in production at scale. Built around the open-source Cluster API, the new version of DKP becomes the single, centralized point of control for an organization’s application infrastructure, empowering organizations to more easily deploy, manage, and scale Kubernetes workloads in Day 2 production environments.
 
-![image](../img/dkp_platform2.png)
+![image](../img/platform_diagram_dark.png)
```

Comment:
```suggestion
![DKP Kubernetes Platform Multi Cluster Management](../img/platform_diagram_dark.png)
```

Reply:
Thanks @msdolbey - that'll look a little better!